test/test_prototype_transforms_functional.py

```diff
@@ -138,6 +138,22 @@ def make_one_hot_labels(
         yield make_one_hot_label(extra_dims_)
 
 
+def make_segmentation_mask(size=None, *, max_value=80, extra_dims=(), dtype=torch.uint8):
```

Comment:
1. Why `max_value=80`? Shouldn't that be the max value of the `dtype`?
2. In `make_segmentation_masks` the default dtype is `torch.long`. Should we align the two?

Reply:
max_value is the number of classes in the mask, for example COCO has 81 classes (with BG)

Reply:
Ah, ok. Maybe we should rename it to

```suggestion
def make_segmentation_mask(size=None, *, num_categories=80, extra_dims=(), dtype=torch.uint8):
```

then?